web_development_101/the_basics/how_this_course_will_work.md

```diff
@@ -1,29 +1,29 @@
+We will start by getting familiar with the internet and your own computer.  Next, we'll work on setting up a development environment and learning about Git and GitHub. Then, we'll go over the basics of front end technologies like HTML, CSS, and Javascript before stepping into the back end with a brief foray into Ruby and Ruby on Rails.  We will finish by learning about pair programming so that you can team up to code with others around the world.
 
-By the end of this unit, you should not only understand how the web works, but be able to identify and differentiate each of the technologies that you will be learning in order to build web applications of your own.  You will be able to build a simple webpage, style it, and add minor elements of interactivity while working comfortably from the command line.
+By the end of this unit, you should not only understand how the web works but also be able to identify and differentiate between each of the technologies that you will be learning in order to build your own web applications.  You will be able to build a simple webpage, style it, and add minor elements of interactivity while working comfortably from the command line.
 
-We will cover a very broad area of knowledge here and that's intentional. It's silly to go diving straight into server-side programming without having a context for what it is and why it's useful (and why you should learn it!).  
+This section intentionally covers a very broad range of knowledge. It's silly to go diving straight into server-side programming without having a context for what it is and why it's useful (and why you should learn it!).  
 
-### How it Works
+### How It Works
 
-This curriculum works by pulling together the best content we can find to learn a particular topic.  In each lesson, we'll introduce the topic and try to provide some useful context before pointing you to external resources made by others.
+This curriculum works by pulling together the best content we can find across the internet for learning a particular topic.  In each lesson, we'll introduce the topic and try to provide some useful context before pointing you to external resources made by others.
 
-Most lessons will contain questions that you should be able to answer before moving on.  Some of the lessons will ask you to do exercises and there will even be several projects to help you grow your understanding by actually building things.  
+Most lessons will contain questions that you should be able to answer before moving on.  Some of the lessons will ask you to do exercises, and we also provide several projects to help you grow your understanding by actually building things.  
```

Comment:
```suggestion
Most lessons will contain questions that you should be able to answer before moving on.  Some of the lessons will ask you to do exercises. We also provide several projects to help you grow your understanding by actually building things.  
```

Reply:
I made some additional changes to help improve the flow.